pkg/cmd/backtest.go

```diff
@@ -677,6 +670,10 @@ func createSymbolReport(
 	return &symbolReport, nil
 }
 
+func n(v float64) fixedpoint.Value {
```

Comment:
remove this function. 
Is this really necessary?

Reply:
gonna work on it this weekend keep sorry for delay